.circleci/config.yml

```diff
@@ -43,6 +44,10 @@ jobs:
             echo "Website added to IPFS: https://ipfs.io/ipfs/$hash"
 
             if [ "$CIRCLE_BRANCH" == "master" ] ; then
+              dnslink-dnsimple -d $DEV_DOMAIN -r _dnslink -l /ipfs/$hash
+            fi
+
+            if [ ! -z "$CIRCLE_TAG" ] ; then
```

Comment:
Nice! I prefer this to using a deploy branch, as is done in peer pad. Would you be up for updating the example in the ipfs-dns-deploy readme with this?

Reply:
Done! https://github.com/ipfs-shipyard/ipfs-dns-deploy/pull/4/files

Reply:
hmm we dont want any old tag to be interpreted as a release... the solution here uses a regex filter to ensure that only tags that look like semver trigger builds... https://discuss.circleci.com/t/migrating-tag-based-deployment-to-2-0/23967/3

What do you think.

Reply:
Updated to use [`semver`](https://www.npmjs.com/package/semver), which is a small package that checks if the input is a sem version! It's easier and more readable than regexing in the shell

Reply:
Perhaps it could be installed globally on the Docker image, just like `dnslink-cloudflare`.

Reply:
Ping @olizilla.

Reply:
Very cool! Let's use npx for now.